actions/ci-lint-go/action.yml

```diff
@@ -127,9 +130,9 @@ runs:
         # use-go-cache set to true
         skip-pkg-cache: ${{ inputs.use-go-cache }}
         skip-build-cache: ${{ inputs.use-go-cache }}
-
         version: ${{ inputs.golangci-lint-version }}
         args: ${{ inputs.golangci-lint-args }}
+        working-directory: ${{ inputs.go-directory }}
```

Comment:
Should the working directory also be applied to the shell step?

Reply:
What about the `upload-artifact` step?

Reply:
good catch!